tests/test_logging.py

```diff
@@ -1430,6 +1432,8 @@ def my_task():
         if record.message == "foo":
             break
 
+    assert record.filename == os.path.basename(__file__)
+    assert record.funcName == inspect.stack()[0][3]
```

Comment:
Probably better to write the literal name of the expected function here, i.e. `"test_patch_print_writes_to_logger_with_task_run_context"` so it's easier to see what the expected value is.

Reply:
@madkinsz Done please have a look